versioned_docs/version-8.0/components/integration-framework/connectors/out-of-the-box-connectors/aws-sqs.md

```diff
@@ -0,0 +1,72 @@
+---
+id: aws-sqs
+title: Amazon Simple Queue Service Connector
+sidebar_label: Amazon SQS Connector
+description: Send messages to Amazon Simple Queue Service from your BPMN process.
+---
+
+The Amazon SQS Connector allows you to connect your BPMN service with Amazon Web Service's [Simple Queue Service](https://aws.amazon.com/sqs/).
```

Comment:
```suggestion
---
id: aws-sqs
title: Amazon Simple Queue Service Connector
sidebar_label: Amazon SQS Connector
description: Send messages to Amazon Simple Queue Service (SQS) from your BPMN process.
---

The Amazon SQS Connector allows you to connect your BPMN service with [Amazon Simple Queue Service (SQS)](https://aws.amazon.com/sqs/).
```

Looking at how Amazon talks about this product on their [product page](https://aws.amazon.com/sqs/), I made the above suggestions.